lessons/Data-Science/AI/ai-decision-trees/langs/en-us/index.adoc

```diff
@@ -0,0 +1,335 @@
+
+
+
+== Decision Trees from Training Datasets @duration{25 minutes}
+
+=== Overview
+
+Students build a decision tree that predicts whether different individuals will purchase a video game or not.
+
+=== Launch
+
+We have already built some extremely simple decision trees. We have seen instances where these trees classify successfully, but we also know that decision trees can sometimes fail when we provide them with ambiguous or unknown data.
+
+We have *not* yet learned about how decision trees built from large datasets often make relatively accurate predictions, recommendations, and diagnoses!
+
+The key to building an effective decision tree is to decide—at every level and every node—which attributes are the most informative ones to ask questions about.
+
+...but how do we make such decisions?!
+
+It turns out, there's an algorithm for that, and it's relatively straightforward.
+
+
+=== Investigate
+
+Have you ever done some online shopping—say, for a new pair of sneakers—only to discover that, for the next several days, you encounter _advertisements for sneakers_ lurking in every corner of the internet that you visit?!
+
+Is it a coincidence? No. On a computer, 'cookies' are small data files that a website can store on your device. These can be used to remember where you were the last time you visited a site, or a setting that you changed and want to keep the next time you visit a site. One particular kind of cookie, the tracking cookie, allows AI designed for marketing to use your individual browsing habits to decide which ads you will be the most susceptible to.
+
+We're going to create a decision tree that predicts whether or not different customers at a particular online store will purchase a video game or not. To do so, we must first train the computer! We will use a training dataset that characterizes 14 different shoppers and then indicates whether each one purchases a video game or not.
+
+@QandA{
+@Q{With your partner, look over the @handout{decision-tree-data.adoc, Training Dataset}. What do you Notice? What do you wonder?}
+@A{Possible responses:}
+@A{Individuals in their twenties always buy the video game.} 
+@A{There are only three new customers; two out of three times, new customers buy the video game.}
+@Q{Can you foresee any problems with making a decision tree based on this dataset? If so, what are they?}
+@A{Responses will vary.}
+}
+
+One problem with this dataset is that _age is continuous_. That won’t work! We need to break these ages down into different groups - which will become different _branches_ that grow out of a _decision node_. *For now, let’s agree to create three groups: teenagers; twenties; and thirties.*
+
+@lesson-instruction{
+- We will complete @printable-exercise{level-1.adoc} together, starting with "age" as the root node.
+- As we create the first level of our tree, you will discover a tool that we use to complete all the levels of our model: the decision stump.
+}
+
+@teacher{
```

Comment:
Part of the conversation inside of this teacher note belongs in the student-facing lesson plan, earlier in this lesson.